# A collection of the basic metadata for a single object detected by YOLO

string object_label
float64 x_left
float64 x_right
float64 y_top
float64 y_bottom
float64 confidence
